packages/neuron-wallet/src/database/chain/ormconfig.ts

```diff
@@ -133,28 +144,55 @@ const connectOptions = async (genesisBlockHash: string): Promise<SqliteConnectio
       IndexerTxHashCacheRemoveField1701234043431,
       CreateCellLocalInfo1701234043432,
     ],
+    subscribers: [
+      AddressSubscribe,
+      AssetAccountSubscribe,
+      MultisigConfigSubscribe,
+      SudtTokenInfoSubscribe,
+      TxDescriptionSubscribe,
+    ],
     logger: 'simple-console',
     logging,
     maxQueryExecutionTime: 30,
   }
 }
 
-export const initConnection = async (genesisBlockHash: string) => {
+const initConnectionWithType = async (genesisBlockHash: string, connectionName: ConnectionName) => {
   // try to close connection, if not exist, will throw ConnectionNotFoundError when call getConnection()
   try {
-    await getConnection().close()
+    await getConnection(connectionName).close()
   } catch (err) {
     // do nothing
   }
-  const connectionOptions = await connectOptions(genesisBlockHash)
+  const connectionOptions = await connectOptions(genesisBlockHash, connectionName)
 
   try {
     await createConnection(connectionOptions)
-    await getConnection().manager.query(`PRAGMA busy_timeout = 3000;`)
-    await getConnection().manager.query(`PRAGMA temp_store = MEMORY;`)
+    await getConnection(connectionName).manager.query(`PRAGMA busy_timeout = 3000;`)
+    await getConnection(connectionName).manager.query(`PRAGMA temp_store = MEMORY;`)
   } catch (err) {
     logger.error(err.message)
   }
 }
 
+export async function initConnection(genesisBlockHash: string) {
+  await initConnectionWithType(genesisBlockHash, 'full')
+  await initConnectionWithType(genesisBlockHash, 'light')
+}
+
+export function migrateDBFile(genesisBlockHash: string) {
+  const originDBFile = dbPath(genesisBlockHash, 'cell')
+  const currentFullDBFile = dbPath(genesisBlockHash, 'full')
+  const currentLightDBFile = dbPath(genesisBlockHash, 'light')
+  if (fs.existsSync(originDBFile) && !fs.existsSync(currentLightDBFile)) {
+    if (!fs.existsSync(currentFullDBFile)) {
+      fs.copyFileSync(originDBFile, currentFullDBFile)
+    }
+    if (!fs.existsSync(currentLightDBFile)) {
```

Comment:
This condition is covered above `if (fs.existsSync(originDBFile) && !fs.existsSync(currentLightDBFile))`